libutils/file_lib.c

```diff
@@ -413,6 +408,7 @@ int safe_open(const char *pathname, int flags, ...)
         return -1;
     }
 
+    size_t final_size = (size_t) -1;
```

Comment:
No because I don't want negative numbers, I just want a "variable is uninitialised" flag.

Furthermore I'm using unsigned since it converts directly from `off_t` which is also unsigned. Hmmm maybe using directly `off_t` would be the best.